demo/compose.simulation.yaml

```diff
@@ -0,0 +1,28 @@
+x-net-config:
+  &net-config
+  network_mode: host
+  ipc: host
+  env_file: net.env
+
+x-gpu-config:
+  &gpu-config
+  runtime: nvidia
+  environment:
+    - DISPLAY=${DISPLAY:?err}
+    - NVIDIA_VISIBLE_DEVICES=all
+    - NVIDIA_DRIVER_CAPABILITIES=all
+
+x-cpu-config:
+  &cpu-config
+  environment:
+    - DISPLAY=${DISPLAY:?err}
+    - LIBGL_ALWAYS_SOFTWARE=1
+
+services:
+  rosbot:
+    image: husarion/rosbot-gazebo:humble-add-simulation-and-demo
```

Comment:
it will be necessary to change it, i think `husarion/rosbot-gazebo:humble` will be the best